SFS_binaural_synthesis/ir_generic.m

```diff
@@ -85,7 +85,9 @@
     % If needed interpolate the given impulse response set and weight, delay the
     % impulse for the correct distance
     ir = get_ir(sofa,X,[phi 0],x0(ii,1:3),'cartesian',conf);
-
+    % Correct Green's function amplitude
+    ir = ir/(4*pi);
```

Comment:
I agree that we need this 1/(4*pi) term for `dummy_irs()`, but do we really need it for measured HRTFs?
There the correct amplitude is supposed to be part of the measurement, isn't it?